SPECS/shadow-utils/shadow-utils.spec

```
@@ -150,6 +150,7 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %post
 %{_sbindir}/pwconv
 %{_sbindir}/grpconv
+chmod 000 etc/shadow
```

Recommend using the macro here
```suggestion
chmod 000 %{_sysconfdir}/shadow
```